CODE_OF_CONDUCT.md

```diff
@@ -1,29 +1,30 @@
-# 📰 CODE OF CONDUCT
-   
+# 📰 Code of Conduct
```

Comment:
We can't make the heading in small characters in this page, as on each page we have headings in all Capital letters.